open_dread_rando/custom_door_types.py

```diff
@@ -63,20 +104,27 @@ def add_weakness(self, weakness: str, new_template: dict):
         }
 
         life_funcs.append(new_func)
+    
+    def change_collision(self, new_template: dict):
```

Comment:
rather than having the check outside the call to this function, I'd start this function with a simple
```py
if self.data.collision is None:
    return
```